layouts/partials/webinar-list-item.html

```diff
@@ -40,8 +40,8 @@
             </div>
         </div>
     </article>
-    {{ if eq .Params.featured true }}
+    <!-- {{ if eq .Params.featured true }}
```

Comment:
... or just `None at this time.` or the like. Rather than comment the code, could we instead just write some code to determine whether to show the heading, etc.?

Reply:
I ended up just removing the `featured` flag from the webinar, so none of the headings show up when there is only one webinar.